lib/cilksan/CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ set(CILKSAN_SOURCES
 
 include_directories(..)
 
-set(CILKSAN_CFLAGS ${SANITIZER_COMMON_CFLAGS} -std=c++11)
+set(CILKSAN_CFLAGS ${SANITIZER_COMMON_CFLAGS})
+
+append_list_if(COMPILER_RT_HAS_STD_CXX11_FLAG -std=c++11 CILKSAN_CFLAGS)
```

Comment:
std=c++11 not c11 for a cflags?

Reply:
Cilksan is a C++ tool, so we require std=c++11, not c11.  For other tools in this repository (e.g., ASAN) the internal CMake variable storing the compiler flags seems to be "*CFLAGS", regardless of whether the project is written in C or C++.  The `append_list_if()` on line 18 does properly check for the C++11 flag, rather than the C11 flag.

Reply:
Odd, but right solution then